07-regression1.Rmd

```diff
@@ -202,7 +217,6 @@ now. Note that for the remainder of the chapter we'll be working with the
 entire Sacramento data set, as opposed to the smaller sample of 30 points above.
 
 ```{r 07-test-train-split}
-set.seed(1234)
 sacramento_split <- initial_split(sacramento, prop = 0.6, strata = price)
```

Comment:
In the last chapter we used a split proportion of 0.75, and here we switch to 0.6 without stating why. We should either state why, or use 0.75 to be consistent.

Reply:
done